core/json/src/main/scala/net/liftweb/json/Formats.scala

```diff
@@ -32,6 +32,14 @@ trait Formats { self: Formats =>
   val fieldSerializers: List[(Class[_], FieldSerializer[_])] = Nil
 
   /**
+   * Support for the experimental tuple decomposition/extraction that represents tuples as JSON
+   * arrays. This provides better support for heterogenous arrays in JSON, but enable it at your
+   * own risk as it does change the behavior of serialization/deserialization and comes
+   * with some caveats (such as Scala primitives not being recognized reliably during extraction).
```

Comment:
Also, should we really label this “experimental”? We're not planning on iterating on it really, it's just a different serialization approach? Perhaps `serializeTuplesAsArrays`?